SharpPcap/WinDivert/WinDivertDevice.cs

```diff
@@ -158,15 +182,6 @@ public ulong GetParam(WinDivertParam param)
 
         public void SendPacket(RawCapture p)
         {
-            if (p is WinDivertCapture packet)
-            {
-                WinDivertAddress address = default;
-                address.IfIdx = packet.InterfaceIndex;
-                address.SubIfIdx = packet.SubInterfaceIndex;
-                address.Flags = packet.Flags;
```

Comment:
This logic existed to allow user to specify `InterfaceIndex` and `Flags` (like for example if packet is inbound or outbound).
The use case for this is a bit limited, but may still be needed.

Reply:
> Can’t they still do that via the send packet variant that accepts a windivertaddress?

It's private

Reply:
Hmm. I looked at making it public but the WinDivertAddress struct is private. Thoughts on how we can improve the api?

Reply:
Ideas:
* We could add `ICaptureHeader` as an optional argument to `SendPacket(header, data)`
* We could add `ICaptureHeader` as a property to `RawCapture`